src/qiskit_sphinx_theme/assets/styles/_admonitions.scss

```diff
@@ -45,92 +44,81 @@ p.topic-title {
   border-bottom-width: 1px;
   position: relative;
   padding-left: 12px;
-}
-.admonition::before {
-  content: '';
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  width: 3px;
-}
 
-.admonition.attention > .admonition-title::before,
-.admonition.caution > .admonition-title::before,
-.admonition.important > .admonition-title::before,
-.admonition.warning > .admonition-title::before {
-  // Rather than using mask-image like Furo normally does, we set `content` to the SVG. This is
-  // because Carbon requires that we have a black exclamation mark for accessibility: the
-  // light yellow background wouldn't be distinct enough compared to the dark yellow icon. Mask
-  // image doesn't let us set colors, only whether to use the background color of the underlying
-  // HTML element vs. use the background color of the SVG container. So, having a black exclamation
-  // mark would require setting the entire .admonition-title background color to black.
-  content: var(--icon-warning);
-  -webkit-mask-image: unset;
-  mask-image: unset;
-  background-color: var(--qiskit-color-admonition-warning--background);
-}
-.admonition.attention > .admonition-title,
-.admonition.caution > .admonition-title,
-.admonition.important > .admonition-title,
-.admonition.warning > .admonition-title {
-  background-color: var(--qiskit-color-admonition-warning--background);
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    width: 3px;
+  }
 }
+
 .admonition.attention,
 .admonition.caution,
 .admonition.important,
 .admonition.warning {
```

Comment:
It cannot because the rule needs to be `.admonition > .admonition-title`. This rule is for the "default" admonition, since Sphinx lets you define your own custom ones.